docs/notebooks/thomson_fitting.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>scape -&gt; scale</p><p>EPW -&gt; Electron Plasma Wave - it's clear from the next code cell, but I'd rather spare people like me the searching! ;)</p><p> </p><p>Could also be a good idea to note that the noise we're adding is uniform and not Gaussian etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>